api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticsearchService.scala

```diff
@@ -101,11 +105,23 @@ class ElasticsearchService @Inject()(elasticClient: ElasticClient)(
       case _ => None
     }
 
+    val sort = queryOptions.sort flatMap {
+      case _: ProductionDateFromSortRequest =>
+        Some(FieldSort("production.dates.range.from"))
+      case _: ProductionDateToSortRequest =>
+        Some(FieldSort("production.dates.range.to"))
+      case _ => None
```

Comment:
Does this allow the user to specify ascending / descending sort?

Reply:
Not yet - see the description:

> sortBy will need to be added next.

Reply:
Ah sorry missed that